src/csharp/Microsoft.Spark.Experimental/Sql/RelationalGroupedDatasetExtensions.cs

```diff
@@ -36,7 +37,7 @@ public static class RelationalGroupedDatasetExtensions
         public static DataFrame Apply(
             this RelationalGroupedDataset dataset,
             StructType returnType, 
-            Func<RecordBatch, RecordBatch> func)
+            Func<FxDataFrame, FxDataFrame> func)
```

Comment:
Do we have concerns about breaking this API?

Reply:
Yes, can we keep this side by side to give users some time for migration to new APIs?

Reply:
Done.